spec/observables/dom/ajax-spec.ts

```diff
@@ -144,7 +144,7 @@ describe('ajax', () => {
 
     it('should send the cookie cross-domain with a custom header when withCredentials is set', () => {
       const obj: AjaxConfig = {
-        url: 'https://some.subresouce.net/some/page',
+        url: 'https://some.subresource.net/some/page',
```

Comment:
Seems harmless.

Insert warning about using domain names you don't control. There are rfc example domains available...